content/networks.md

```diff
@@ -0,0 +1,6 @@
+---
+id: Networks
+title: Meet Our Participants
```

Comment:
I am not sure if the Title should not be the same as in the navbar. Maybe we could have this sentence as a subtitle

Reply:
Done, thanks!